src/librustc_codegen_ssa/traits/backend.rs

```diff
@@ -52,8 +52,7 @@ pub trait ExtraBackendMethods: CodegenBackend + WriteBackendMethods + Sized + Se
         &self,
         tcx: TyCtxt<'_>,
         cgu_name: Symbol,
-        tx_to_llvm_workers: &mpsc::Sender<Box<dyn std::any::Any + Send>>,
-    );
+    ) -> (ModuleCodegen<Self::Module>, u64);
```

Comment:
Please add a comment explaining that the `u64` in the return tuple is the estimated compilation cost of the CGU.